Formula/git-sizer.rb

```diff
@@ -0,0 +1,30 @@
+class GitSizer < Formula
+  desc "Compute various size metrics for a Git repository"
+  homepage "https://github.com/github/git-sizer"
+  url "https://github.com/github/git-sizer/archive/v1.0.0.tar.gz"
+  sha256 "8ef6b2adb646ccb72f5ccfebd19a57371b119da1aac02ef06f1a46af6f7447a0"
+
+  head do
+    url "https://github.com/github/git-sizer.git"
```

Comment:
Single line, `head "https://github.com/github/git-sizer.git"`, after `sha256` with no blank line in-between

Reply:
:recycle: Implemented in 8725b5e4bb.